database.go

```diff
@@ -12,9 +12,9 @@ func loadDatabase(dbFile string) error {
 	}
 	statement := `
 		CREATE TABLE IF NOT EXISTS comments (
-			url text not null,
-			name text not null,
-			comment text not null,
+			url varchar(2083) not null,
+			name varchar(200) not null,
+			comment varchar(10000) not null,
```

Comment:
10000 comment limit is pretty good IMO. Someone posted the first hundred thousand digits of pi on the demo site :D

And I'm yet to meet someone with a name longer than 200 chars :P